node_modules/read-package-tree/rpt.js

```diff
@@ -94,7 +94,7 @@ Link.prototype = Object.create(Node.prototype, {
 Link.prototype.target = null
 Link.prototype.realpath = ''
 
-function loadNode (logical, physical, cache, cb) {
+function loadNode (logical, physical, cache, packageFilename, cb) {
```

Comment:
Very unfortunate that changes to this package also need to be made. Not sure how you folks have handled this type of dep change in the past -- but since they are checked in, monkey patching seemed like the way to go at the time. 

Any suggestions for this?

Reply:
Monkey patching things in node_modules is never something we do. We patch the other module, do a release there, then update as a part of the commit.

While figuring out what changes to make it's fine though, it just can't be merged like that.